server/src/main/java/org/apache/druid/server/initialization/jetty/JettyServerModule.java

```diff
@@ -475,6 +478,25 @@ public void stop()
         Lifecycle.Stage.SERVER
     );
 
+    if (config.isDefaultJettyErrorResponseInJson()) {
```

Comment:
Nit: Create ErrorHandler (depending on how configs are set outside) of server.setErrorHandler to avoid repeating code.